<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* ---- 设置 centralwidget 的背景图片 ---- */
QWidget#centralwidget {
    border-image: url(:/backgrounds/bg_main_window.jpg) 0 0 0 0 stretch stretch;
}

/* ---- 欢迎语标签 ---- */
QLabel#welcomeLabel, QLabel#patientNameLabel {
    border-image: none; /* 取消背景图片继承 */
    background-color: transparent; /* 设置透明背景 */
    font-size: 16px;
    color: #333; /* 在亮色背景上，深色文字更清晰 */
}
QLabel#patientNameLabel {
    font-weight: bold;
    color: #000; /* 姓名用更深的颜色突出 */
}

/* ---- 核心功能按钮 (我们没有这个按钮了，可以删除或保留) ---- */
/*
QPushButton#mainAppointmentButton { ... }
*/

/* ---- 功能网格按钮 (QToolButton) ---- */
QToolButton {
    background-color: rgba(255, 255, 255, 0.85); /* 半透明白色背景，透出背景图 */
    color: #2c3e50; /* 深蓝灰色文字 */
    border: 1px solid rgba(0, 0, 0, 0.1); /* 半透明的细边框 */
    border-radius: 12px; /* 更大的圆角，更可爱 */
    padding: 10px;
    font-size: 13px;
    font-weight: bold;
    border-image: none; /* 取消背景图片继承 */
}
QToolButton:hover {
    background-color: rgba(255, 255, 255, 1); /* 悬浮时不透明 */
    border: 1px solid #3498db; /* 主题蓝色边框 */
}
QToolButton:pressed {
    background-color: #ecf5ff; /* 按下时淡蓝色背景 */
}

/* ---- 提示语标签 ---- */
QLabel#tipsLabel {
    font-size: 13px;
    color: #2c3e50;
    background-color: rgba(255, 255, 255, 0.7); /* 半透明白色背景 */
    border: 1px solid rgba(0, 0, 0, 0.05);
    border-radius: 8px;
    padding: 10px;
    margin-left: 15px;
    margin-right: 15px;
    margin-top: 5px;
    border-image: none; /* 取消背景图片继承 */
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="welcomeLabel">
        <property name="text">
         <string>欢迎您</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="patientNameLabel">
        <property name="text">
         <string>患者姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="tipsLabel">
      <property name="styleSheet">
       <string notr="true">/* ---- 提示语标签 ---- */
QLabel#tipsLabel {
    font-size: 13px;
    color: #2c3e50;
    background-color: rgba(255, 255, 255, 0.7); /* 半透明白色背景 */
    border: 1px solid rgba(0, 0, 0, 0.05);
    border-radius: 8px;
    padding: 10px;
    margin-left: 15px;
    margin-right: 15px;
    margin-top: 5px;
    border-image: none; /* 取消背景图片继承 */
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QToolButton" name="chatButton">
        <property name="text">
         <string>聊天/消息</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_chat.png</normaloff>:/icons/icon_home_chat.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="paymentButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>在线支付</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_wallet.png</normaloff>:/icons/icon_home_wallet.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="patientInfoButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>个人资料</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_psndt.png</normaloff>:/icons/icon_home_psndt.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="doctorInfoButton">
        <property name="text">
         <string>医生信息</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_doctor3.png</normaloff>:/icons/icon_home_doctor3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="emrButton">
        <property name="text">
         <string>电子病历</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_mdcrcd.png</normaloff>:/icons/icon_home_mdcrcd.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="aiButton">
        <property name="text">
         <string>ai诊断</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_ai1.png</normaloff>:/icons/icon_home_ai1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="myAppointmentsButton">
        <property name="text">
         <string>我的预约</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_list.png</normaloff>:/icons/icon_home_list.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="consultationButton">
        <property name="text">
         <string>预约挂号</string>
        </property>
        <property name="icon">
         <iconset resource="patientclient.qrc">
          <normaloff>:/icons/icon_home_calender.png</normaloff>:/icons/icon_home_calender.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="patientclient.qrc"/>
 </resources>
 <connections/>
</ui>
